create_infotexts.py

```diff
@@ -43,6 +43,31 @@ def load_places_mapping():
     return places_dict
 
 
+def load_keywords_mapping():
+    """
+    Read wikitable html into Pandas DataFrame, transform it and return a dictionary.
+    
+    :return: dictionary
+    """
+    kw_maps_url = "https://commons.wikimedia.org/wiki/Commons:Medelhavsmuseet/batchUploads/Cypern_keywords"
+    keywords = pd.read_html(kw_maps_url, attrs={"class": "wikitable sortable"}, header=0)
+    kw1 = keywords[0] # First is the actual column <Nyceklord>
+    
+    kw1 = kw1.set_index("Nyckelord")
+    kw1 = kw1[["Commons category", "wikidata"]] # Remove col "fredquency" for boolean filtering to work
+    
+    kw1[kw1 == "-"] = None  # Pandas "boolean indexing" kung-fu to replace all "-" with None
+    
+    kw_dict = {}
+    for index, row in kw1.iterrows():
+        kw_dict[index] = {}
+        if row["Commons category"]:
+            kw_dict[index]["commonscat"] = row["Commons category"]
+        kw_dict[index]["wikidata"] = row["wikidata"] # should always be present
```

Comment:
Ah that is because you are still doing
```python
if row["commonscat"]:
    places_dict[index]["commonscat"] = row["commonscat"]
```
on L39
If you change that to ```places_dict[index]["commonscat"] = row["commonscat"]``` then you don't need the `.get()`.

But it's just about consistency so not a blocker.